agent/nrpe.rb

```diff
@@ -43,38 +57,49 @@ def self.run(command)
         return 3, "No such command: #{command}" unless nrpe_command
 
         output = ""
-        shell = Shell.new(nrpe_command[:cmd], {:stdout => output, :chomp => true})
+        shell = Shell.new(nrpe_command, {:stdout => output, :chomp => true})
```

Comment:
This seems incorrect, `plugin_for_command` returns `nil` or `{ :cmd => plugin}`.

If `plugin_for_command` were to return `nil` or `plugin` this edit would make sense, but otherwise we're passing a Hash as the first argument to ::MCollective::Shell.new which isn't going to work out well.